include/http/request_handler.js

```diff
@@ -164,9 +164,9 @@ module.exports = function RequestHandlerModule(pb) {
           uid: site.uid,
           displayName: site.displayName,
           hostname: site.hostname,
-          defaultLocale: site.defaultLocale,
-          supportedLocales: site.supportedLocales,
-          prevHostnames: site.prevHostnames
+          defaultLocale: site.defaultLocale || pb.Localization.getDefaultLocale(),
+          supportedLocales: site.supportedLocales || [],
```

Comment:
Is there something else the defaultLocale should resolve to?


- Wes Anderson


-------- Original message --------
From: Brian <notifications@github.com>
Date: 01/05/2016 2:57 PM (GMT-05:00)
To: pencilblue/pencilblue <pencilblue@noreply.github.com>
Cc: Wes Anderson <Wes.Anderson@careerbuilder.com>
Subject: Re: [pencilblue] Load site defaults (#918)


In include/http/request_handler.js<https://github.com/pencilblue/pencilblue/pull/918#discussion_r48888838>:

> @@ -164,9 +164,9 @@ module.exports = function RequestHandlerModule(pb) {
>            uid: site.uid,
>            displayName: site.displayName,
>            hostname: site.hostname,
> -          defaultLocale: site.defaultLocale,
> -          supportedLocales: site.supportedLocales,
> -          prevHostnames: site.prevHostnames
> +          defaultLocale: site.defaultLocale || pb.Localization.getDefaultLocale(),
> +          supportedLocales: site.supportedLocales || [],


@wesanderson303<https://github.com/wesanderson303> Are you sure this works as expected? I think this will always default to English because there are no other allowed locales for the site

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/918/files#r48888838>.